lang/c++/impl/Compiler.cc

```diff
@@ -384,7 +396,7 @@ static NodePtr makeEnumNode(const Entity &e,
 
 static NodePtr makeFixedNode(const Entity &e,
                              const Name &name, const Object &m) {
-    int v = static_cast<int>(getLongField(e, m, "size"));
+    size_t v = getLongField(e, m, "size");
```

Comment:
In modern C++ we try to avoid explicitly specifying type in situations like this: please use `auto`.

Reply:
updated

Reply:
updated